src/cljs/ataru/virkailija/application/application_search_control_handlers.cljs

```diff
@@ -15,6 +16,19 @@
    (assoc-in db show-path :complete)))
 
 (reg-event-db
+ :application/show-search-ssn
+ (fn [db [_ _]]
```

Comment:
Can you just entirely omit the second parameter here? I think reframe might support that.